subxt/src/utils/mod.rs

```diff
@@ -4,13 +4,23 @@
 
 //! Miscellaneous utility helpers.
 
+pub mod bits;
+
 use codec::{
     Decode,
     DecodeAll,
     Encode,
 };
 use derivative::Derivative;
 
+pub use bits::{
+    BitOrder,
+    BitStore,
+    DecodedBits,
+    Lsb0,
+    Msb0,
+};
```

Comment:
Nitpick: imo I wouldn't expose these here; make people go via `utils::bits` for clarity :)